spark-connector/build.gradle.kts

```diff
@@ -14,26 +15,35 @@ repositories {
 
 val scalaVersion: String = project.properties["scalaVersion"] as? String ?: extra["defaultScalaVersion"].toString()
 val sparkVersion: String = libs.versions.spark.get()
+val sparkMajorVersion: String = extra["sparkMajorVersion"].toString()
```

Comment:
SparkMajorVersion should be calculated from SparkVersion.

Reply:
done